lib/elastic_apm/metadata/cloud_info.rb

```diff
@@ -97,13 +97,12 @@ def fetch_gcp
         zone = metadata["instance"]["zone"]&.split("/")&.at(-1)
 
         self.provider = "gcp"
-        self.instance_id = metadata["instance"]["id"]
+        self.instance_id = metadata["instance"]["id"].to_s
```

Comment:
The [issue description](https://github.com/elastic/apm/issues/826) says to check that the int is being handled correctly but doesn't say it should be collected as a string. Is using a string the solution?

Reply:
This was a bug in a the Ruby impl unrelated to the spec change for https://github.com/elastic/apm/issues/826.

The spec had already said:

> `instance.id` as a string

and the intake schema requires a string (https://github.com/elastic/apm-agent-ruby/blob/8d4a30cd3f592d6fcb7fc97826d71ec6d908b4b7/spec/fixtures/metadata.json#L52-L57).